go/webpack/cbor_format_test.go

```diff
@@ -27,6 +42,17 @@ func hpackByteArray(headersAndValues ...string) []byte {
 	return append(cbor.Encoded(cbor.TypeBytes, len(result)), result...)
 }
 
+func hpackDecode(t *testing.T, encoded []byte) HTTPHeaders {
```

Comment:
is it more canonical to name the return value than to declare one in the first line of the function?